app/views/users/edit.html.slim

```diff
@@ -1,8 +1,8 @@
 - if params[:welcome]
   h1 Welcome!
 
-  p Please complete your profile:
-
+  p Please <strong>check your email address</strong> and fill all required fields to complete signing up.
```

Comment:
Should we notify users that updating their email address will require confirming, or will updating the email address reset the `confirmed` status?

Reply:
👍 The notification should be in the (yellow) notice that should appear after they submit the form. Updating the email address will not reset the `confirmed` status, but will set the `unconfirmed_email`.

Currently the notification only appears for _un_confirmed users, so I'll fix that! Thanks!

Reply:
No wait, that cannot be solved yet. See my general comment below.